.github/workflows/publish.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: GoReleaser
         uses: goreleaser/goreleaser-action@v2
         with: 
-          version: v0.155.2
+          version: latest
```

Comment:
In #39, which was around the time of the Go 1.16 release, we had pinned the version of goreleaser as a workaround for an upstream issue build Apple ARM binaries.